packages/webdriverio/package.json

```diff
@@ -65,6 +65,7 @@
     "chromedriver": "latest",
     "cross-dirname": "^0.1.0",
     "delay": "^5.0.0",
+    "devtools": "^8.27.2",
```

Comment:
In [webdriverio v8.13](https://github.com/webdriverio/webdriverio/blob/v8.13.14/packages/webdriverio/package.json#L81) they had the `devtools` package as a dependency. However in v[8.14 they removed it](https://github.com/webdriverio/webdriverio/blob/v8.14.0/packages/webdriverio/package.json#L95-L97\) and added it as a peer dependency. Their [changelog for v8.14](https://github.com/webdriverio/webdriverio/releases/tag/v8.14.0) lists no such change even though that's a very significant change to no longer auto download a required npm package to run the devtools protocol.